OneLogin.xcodeproj/project.pbxproj

```diff
@@ -1627,8 +1655,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/govuk-one-login/mobile-ios-common.git";
 			requirement = {
-				branch = main;
-				kind = branch;
+				kind = exactVersion;
+				version = 1.1.0;
```

Comment:
love to see it!

Reply:
(until we have a proper versioning strategy in place... in medium term we probably want to pin to latest minor version eg. `^1.1.0`